src/data.extra.json

```diff
@@ -606,6 +606,14 @@
       "min": 0,
       "max": 1000
     },
+    "tx_queue_locals": {
+      "type":"string[]",
+      "name": "Local Accounts",
+      "description": "Specify local accounts for which transactions are prioritized in the queue. ACCOUNTS is a comma-delimited list of addresses.",
```

Comment:
I think we can remove the part `ACCOUNTS is a comma-delimited list of addresses.` since people can just click "+" to add a new account for this config item, in the parity config generator (and then there's no comma to be seen, so let's not confuse users :p)